src/nav/infrastructure.yml

```diff
@@ -157,6 +157,38 @@ pages:
             path: /docs/infrastructure/prometheus-integrations/install-configure-openmetrics/add-rename-or-copy-prometheus-attributes
           - title: Mutual TLS authentication
             path: /docs/infrastructure/prometheus-integrations/install-configure-openmetrics/add-mutual-tls-prometheus-endpoints
+      - title: Install and configure the Prometheus agent
+        pages:
+          - title: Install the Prometheus agent
```

Comment:
as discussed offline we might want to hide some of the docs to be able to release them before GA